lib/chat/views/chatroom/bloc/chat_action_bloc/chat_action_event.dart

```diff
@@ -0,0 +1,205 @@
+part of 'chat_action_bloc.dart';
+
+abstract class ChatActionEvent extends Equatable {}
+
+class PostConversation extends ChatActionEvent {
```

Comment:
why all the classes are added in the same file., plus if it required to keep in same file, add comments?